### ISSUE TITLE
Binary distributions break copyright licensing

### ISSUE THREAD

**Issue by RochaStratovan** [intel/mkl-dnn#596]
# Summary
The Binary distributions break licensing laws because they do not include the licenses of the 3rd party libraries that they use.

# Additional details
The Linux, Windows and macOS distributions are breaking copyright licensing laws because they do not include the text of the 3rd party licenses.

The `LICENSE`  file included with the distributions reference the 3rd party libraries that it uses such as:

>    DNNL includes components with separate copyright
>    notices and license terms.
> 
>    XByak, 3-clause BSD license
>    Copyright (c) 2007 MITSUNARI Shigeo
>    See full copyright notice and license text in src/cpu/xbyak/COPYRIGHT
> 

However the binary distributions do not actually include these 3rd party COPYRIGHT files, which breaks the 3rd Party license. In particular the following text from the COPYRIGHT files.

> Redistributions in binary form must reproduce the above copyright notice,
> this list of conditions and the following disclaimer in the documentation
> and/or other materials provided with the distribution.

In short the entire COPYRIGHT file of the 3rd Party Library needs to be included the binary distribution's LICENSE file or as separate files for the specific libraries.

**Comment by vpirogov** [comment]
Hi @RochaStratovan,

Thank you for your comment. Intel takes the licensing obligations of third party software very seriously. A Github distribution could encompass everything in the particular repository and Intel has provided the third party notice and license information in the repository.  To make it easier for users like you, however, Intel is reassessing this practice.
